stores/filestore.go

```diff
@@ -1112,7 +1112,11 @@ func (ms *FileMsgStore) enforceLimits() error {
 		ms.totalBytes -= firstMsgSize
 
 		// Remove the first message from our cache
-		Noticef("WARNING: Removing message[%d] from the store for [`%s`]", ms.first, ms.subject)
+		if !ms.hitLimit {
+			ms.hitLimit = true
+			Noticef("WARNING: One of the limits (msgs=%v bytes=%v) reached for store [`%s`]",
```

Comment:
How about: "Store limits for ['%s'] store have been exceeded. The store will permanently prune old messages as new messages arrive."